### PR TITLE
Add svg in config-header

### PR DESCRIPTION
**Pull request by siddhant1**

### CODE REVIEW

File: src/components/input-panel/compiled-spec-header/index.tsx [hunk 1]
```diff
@@ -62,7 +62,9 @@ class CompiledSpecDisplayHeader extends React.PureComponent<Props> {
             </li>
           </ul>
 
-          <ChevronDown />
+          <div className="down-button-svg">
```
Comment (domoritz) [inline src/components/input-panel/compiled-spec-header/index.tsx]: Why is this necessary?

Comment (siddhant1) [reply src/components/input-panel/compiled-spec-header/index.tsx]: I tried `.editor-header svg` , its not working for me

Comment (domoritz) [reply src/components/input-panel/compiled-spec-header/index.tsx]: I'm not really satisfied by that answer.

Comment (siddhant1) [reply src/components/input-panel/compiled-spec-header/index.tsx]: svg is an inline-element , we can place it inside div and position the div , otherwise if we want to do it with only SVG tag, we have to position it using pixels

Comment (domoritz) [reply src/components/input-panel/compiled-spec-header/index.tsx]: Really? Can't we just make it a block element for example?

Comment (siddhant1) [reply src/components/input-panel/compiled-spec-header/index.tsx]: Tried! It's actually not working , maybe when we switch we remove some divs!
Np, I am trying to place it without the div , but I think this works good as well!
